client/src/components/graphs/OrganizationalChart.js

```diff
@@ -49,81 +58,132 @@ const GQL_GET_CHART_DATA = gql`
     }
   }
 `
+const transitionDuration = 200
 
 const ranks = Settings.fields.person.ranks.map(rank => rank.value)
 
 const sortPositions = (positions, truncateLimit) => {
   const allResults = [...positions].sort((p1, p2) =>
     ranks.indexOf(p1.person?.rank) > ranks.indexOf(p2.person?.rank) ? -1 : 1
   )
-  return truncateLimit && truncateLimit < allResults.length
+  return truncateLimit !== undefined && truncateLimit < allResults.length
     ? allResults.slice(0, truncateLimit)
     : allResults
 }
 
-export default class OrganizationalChart extends SVGCanvas {
-  static propTypes = {
-    org: PropTypes.object.isRequired
-  }
+const OrganizationalChart = props => {
```

Comment:
Prefer `const OrganizationalChart = ({ org, width, height }) => {` instead of referring to `props.something` everywhere.